packages/web/src/js/utils/ComponentFunctions.ts

```diff
@@ -18,7 +18,7 @@ const enableDataTrigger = (
   EventHandler.on(window, 'DOMContentLoaded', (event: Event) => {
     SelectorEngine.findAll(`[${dataTriggerAttribute}="${name}"]`).forEach((toggleEl) => {
       EventHandler.on(toggleEl, 'click', function handleClick() {
-        const target = getElement(this);
+        const target = getTargetOrElement(getElement(this));
```

Comment:
Koukal jsem k čemu to je a dospěl jsem k závěru, že úplně nevím, jak bych to napsal jinak, ale mít "nerozhodný" název funkce mi přijde při nejmenším divné. Navíc když se pak používá k tomu, že se assignuje do proměnné, která se jmenuje jako jedna z těch "nerozhodných" variant z názvu funkce.

Minimálně mě to přinutilo podívat se blíž do kódu, co to dělá, protože mi to nebylo na první pohled jasné a nedalo mi to.

Reply:
Ok, good point. Please, suggest a better name or structure. The usage for this function is that it should return the main element for the component instance, eg. I do not need to search for the `data-target` element in every component when need to create an instance for it, but it will get it now when it exists. So that is why get data-target or return an element where the toggle is placed.